pkg/properties/properties.go

```diff
@@ -96,3 +96,10 @@ func CreateKeyValuePairs(m map[string]string) string {
 	}
 	return b.String()
 }
+
+func replaceSpecialCharacters(val string) string {
+	// Replace \\n, \\r and \\t with newline, carriage return and tab characters as specified in
+	// https://docs.oracle.com/cd/E23095_01/Platform.93/ATGProgGuide/html/s0204propertiesfileformat01.html.
+	return strings.ReplaceAll(strings.ReplaceAll(
+		strings.ReplaceAll(val, "\\n", "\n"), "\\r", "\r"), "\\t", "\t")
```

Comment:
From the link:
> The backslash character must be escaped as a double backslash.

Seems like we can simplify to the following (and move this out of its own function since it's a 1-liner)

```suggestion
	return strings.ReplaceAll(val, "\\", "\")
```

Reply:
@brianstrauch Thanks for the review. "\" is not a valid string and will cause compilation error. It is an escape character that changes the meaning of the next character.

Do you want still want me to inline the following?
return strings.ReplaceAll(strings.ReplaceAll(
		strings.ReplaceAll(val, "\\n", "\n"), "\\r", "\r"), "\\t", "\t")

Reply:
Oops, updated the suggestion above.

Reply:
strings.ReplaceAll(val, `\\`, `\`) is equivalent to strings.ReplaceAll(val, "\\\\", "\\") which wouldn't help in converting "\\n" to "\n"